CITATION.rst

```diff
@@ -3,20 +3,21 @@ Acknowledging or Citing SunPy
 
 If you have used SunPy in your scientific work we would appreciate it if you would acknowledge it.
 The continued growth and development of SunPy is dependent on the community being aware of the use SunPy.
-If you use SunPy, we therefore ask that you acknowledge SunPy appropriately in a publication, presentation, poster, or talk.
 
--  **For a publication**, we recommend the following line be added to
-   the conclusion or acknowledgements:
+Publications
+------------
+
+Please add the following line to the conclusion or acknowledgements:
 
    *This research has made use of SunPy, an open-source and free
    community-developed solar data analysis package written in Python
    (citation).*
 
-   The citation is to the `SunPy v0.5 paper`_/`arXiv(open access)`_. If
-   the journal allows please also include a link to sunpy.org. If you
```

Comment:
I've got rid of the "linking to sunpy.org" bit to make the whole thing a bit less verbose, but am happy to put it back in.

Reply:
I think it's fine without. We want these to be as easy as possible.